packages/sdk/tasks/one-time/stella-upgrade.ts

```diff
@@ -0,0 +1,89 @@
+import { TransactionReceipt } from "@ethersproject/abstract-provider";
+import { task, types } from "hardhat/config";
+
+import { FuseFeeDistributor } from "../../lib/contracts/typechain/FuseFeeDistributor";
+
+task("plugin:deploy", "Deploy ERC4626 Strategy")
+  .addParam("creator", "Deployer Address", "deployer", types.string)
+  .addOptionalParam(
+    "otherParams",
+    "other params that might be required to construct the strategy",
+    undefined,
+    types.string
+  )
+  .setAction(async (taskArgs, { ethers, deployments }) => {
+    // @ts-ignore
+    const midasSdkModule = await import("../../tests/utils/midasSdk");
+    const sdk = await midasSdkModule.getOrCreateMidas();
+
+    const signer = await ethers.getNamedSigner(taskArgs.creator);
+
+    const fuseFeeDistributor = (await ethers.getContract("FuseFeeDistributor", signer)) as FuseFeeDistributor;
+
+    const erc20PluginRewardsDel = await deployments.deploy("CErc20PluginRewardsDelegate", {
+      from: signer.address,
+      args: [],
+      log: true,
+      waitConfirmations: 1,
+    });
+    console.log("CErc20PluginRewardsDelegate: ", erc20PluginRewardsDel.address);
+
+    const erc20PluginRewardsDelegate = await ethers.getContract("CErc20PluginRewardsDelegate", signer);
+
+    const oldImplementations = [
+      ethers.constants.AddressZero,
+      ethers.constants.AddressZero,
+      ethers.constants.AddressZero,
+    ];
+    const newImplementations = [
+      sdk.chainDeployment.CErc20Delegate.address,
+      sdk.chainDeployment.CErc20PluginDelegate.address,
+      sdk.chainDeployment.CErc20PluginRewardsDelegate.address,
+    ];
+    const arrayOfFalse = [false, false, false];
+    const arrayOfTrue = [true, true, true];
+
+    if (sdk.chainDeployment.CErc20PluginRewardsDelegate.address !== "0x32Be4b977BaB44e9146Bb414c18911e652C56568") {
```

Comment:
no, this is the address of the market delegator/proxy - the correct address of the current `CErc20PluginRewardsDelegate ` is `0xad0538f54d7a503FE138b9fB62711B2034E0FD94 `

Reply:
Good catch